### ISSUE TITLE
Attributes default value not working for label, instead defaults to 1st value in list

### ISSUE THREAD

**Issue by yuminghuang23** [opencv/cvat#6415]
### My actions before raising this issue
- [X] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [X] Searched [past issues](/issues)

Default value for label's attribute is not getting used in cvat, instead it uses the first attribute value in the list. This is during the creation of a project, using code below.
The version of the sdk and cvat are stated below.
Python cvat-sdk version: 2.3.0 
cvat version: 2.2.0

Code sample:
```
labels = [{'attributes': [{'default_value': 'd',
                  'input_type': 'radio',
                  'mutable': True,
                  'name': 'attribute_values',
                  'values': ['a',
                             'b',
                             'c',
                             'd']}],
  'name': 'label_1'},
 {'name': 'label_2'}]

project_write_request = ProjectWriteRequest(
            name="project_name",
            labels=labels,
        )

with ApiClient(self.configuration) as api_client:
    try:
        data, response = api_client.projects_api.create(
            project_write_request, x_organization="organisation_name"
        )
        if response.status not in [200, 201, 202]:
            raise cvat_exceptions.CvatProjectCreationError("some msg...)

        return data, response
    except exceptions.ApiException as e:
        print("Exception when calling creating project: %s\n" % e)
```
Additional info:
I did try to use the latest version of cvat-sdk (2.4.7), but it does not seem to be compatible with cvat 2.2.0 on `api_client.tasks_api.list()` where it gives this error from inside the library: `TypeError: _from_openapi_data() missing 1 required positional argument: 'jobs'`. And updating cvat is also not really ideal in my case.

### Steps to Reproduce (for bugs)

1. Make the above listed labels and create a project
2. Login to cvat ui and check that the projects label detail
3. Sees that the default value is `a` and not `d`

### Expected Behaviour

Expect the default value to be `d` and not the first value in the attribute list. 

### Current Behaviour

Default value is set to the first value of the list, irrespective of the value set in the `default_value` field

### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05): 20.10.23
- Are you using Docker Swarm or Kubernetes? No
- Operating System and version (e.g. Linux, Windows, MacOS): Linux
- Code example or link to GitHub repo or gist to reproduce problem: Listed above
- Python cvat-sdk version: 2.3.0 
- cvat version: 2.2.0
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

**Comment by bsekachev** [comment]
@yuminghuang23 

Wrong API specification, but actually ``default_value`` is read only value (equals to the first value from the list).
Maybe better to change this behaviour..

**Comment by yuminghuang23** [comment]
Yeah, it is a bit confusing and the documentation doesn't really explain
it.

**Comment by bsekachev** [comment]
Related #6474
